TicketSwap/solar
# Introduce Radio button

# Description

Introducing `Radio`, `Label`, `RadioWithLabel` and `RadioGroup` component.

## How to test

- Checkout this branch
- `$ yarn storybook`
- Go to the Radio stories and check if everything is working as expected.

## Screenshots

<img width="118" alt="image" src="https://user-images.githubusercontent.com/12168698/155306748-327bf3a0-622d-4070-a959-f11c34ce550d.png">

<img width="271" alt="image" src="https://user-images.githubusercontent.com/12168698/155306786-a7b65382-e9a7-48cb-9a58-466d047bb5a1.png">

<img width="308" alt="image" src="https://user-images.githubusercontent.com/12168698/155306815-ff12d7b6-0fea-4ca3-8acf-f39bc10581bc.png">

## To be notified

@TicketSwap/frontend 

## Links

Solves a TO-DO item in sirius which you can find [here](https://github.com/TicketSwap/sirius/issues/5708)
